### ISSUE TITLE
[BUG] Repeaters in templatefields is throwing an error

### ISSUE THREAD

**Issue by ryanturner10** [bolt/bolt#5183]
Having trouble saving content in a repeater field inside the templatefields tab.
Any ideas/resolutions?

Doctrine\DBAL\DBALException thrown with message "Unknown column type "repeater" requested. Any Doctrine type that you use has to be registered with \Doctrine\DBAL\Types\Type::addType(). You can get a list of all the known types with \Doctrine\DBAL\Types\Type::getTypesMap(). If this error occurs during database introspection then you might have forgot to register all database types for a Doctrine Type. Use AbstractPlatform#registerDoctrineTypeMapping() or have your custom types implement Type#getMappedDatabaseTypes(). If the type name is empty you might have a problem with the cache or forgot some mapping information."

Details
-------

 - Relevant Bolt Version: [ 3.0b2 ]
 - Install type: [ Composer install ]
 - PHP version: [ 5.6 ]
 - Used webserver: [ Apache ] version [ 2.4.16 ]

**Comment by GawainLynch** [comment]
@ryanturner10 I don't supposed you could include the relevant ContentType from `app/config/contenttypes.yml` by any chance?

**Comment by ryanturner10** [comment]
Of course. It was just a simple 'pages' contentType, nothing out of the ordinary!

```
pages:
    name: Pages
    singular_name: Page
    fields:
        title:
            type: text
            class: large
            group: content
        slug:
            type: slug
            uses: title
        image:
            type: image
        teaser:
            type: html
            height: 150px
        body:
            type: html
            height: 300px
        template:
            type: templateselect
            filter: '*.twig'
    recordsperpage: 100
```

**Comment by GawainLynch** [comment]
It is :beers: time on a Friday, so excuse me if I am being even more stupid than my usual self (which can be pretty spectacularly stupid :laughing:), but where is the repeating fields coming from?

**Comment by ryanturner10** [comment]
No worries man, It was coming out of the theme.yml file in the theme folder as a templatefield!
To be fair I didn't actually specify that when I posted the issue. My bad!

```
# Theme config

# # Google analytics tracking ID
# gaid: UA-XXXXX-X
template_directory: templates
# # template_directory:
# record_template: default.twig
templatefields:
    home.twig:
        whatever_the_sector_slider:
            label: Whatever the Sector Slider
            type: repeater
            fields:
                heading:
                    type: text
                    label: Sector Name
                body:
                    type: textarea
                    label: Sector Info
```

**Comment by SahAssar** [comment]
Seems to have been fixed by #5542 (In other words, "it works for me" ;) ) Please reopen if you are still having issues!
